src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolver/VariadicRequestAttributeResolver.php

```diff
@@ -18,25 +18,27 @@
 /**
  * Grabs the variadic value from the request and returns it.
  *
- * Opposite of {@see ArgumentFromAttribute}.
+ * Opposite of {@see ArgumentFromAttributeResolver}.
```

Comment:
This was mainly a left-over from the return I had in the `ArgumentResolver` which we decided upon to remove it. Now that I look at it, it might be able to be removed. I'd like to keep the check if it's not variadic though, otherwise it might cause an issue as it's triggered before the variadic version and only yields once instead of for all.

While checking the code, I was also thinking about the supports vs resolve in the variadic resolver. right now the resolve checks if the value is an array and if not throws an exception. This makes debugging the issue easier. However, you'd expect the is_array check on the value to be in supports method. What is your opinion on that while we're on the subject?